content/guides/core/automations/automation-events.md

```diff
@@ -0,0 +1,55 @@
+---
+menu:
+  default:
+    identifier: automation-scopes
+    parent: automations
+title: Automation events and scopes
+weight: 2
+---
+{{% pageinfo color="info" %}}
+{{< readfile file="/_includes/enterprise-only.md" >}}
+{{% /pageinfo %}}
+
+An automation can run when a specific event occurs at a given scope, either a registry or a project. This page describes the events that can trigger an automation at each scope.
+
+Learn more about automations in the [Automations overview]({{< relref "/guides/core/automations/" >}}) or [Create an automation]({{< relref "create-automations/" >}}).
+
+## Registry
+This section describes the scopes and events for an automation in a [Registry]({{< relref "/guides/core/registry/">}}).
+
+1. Navigate to the **Registry** App at https://wandb.ai/registry/.
+1. Click the name of a registry, then view and create automations in the **Automations** tab.
+
+Learn more about [creating automations]({{< relref "create-automations/" >}}).
+
+### Scopes
+You can create a Registry automation at these scopes:
+- A [Registry]({{< relref "/guides/core/registry/">}}). The automation watches for the event taking place on any collection in the registry.
+- A single collection in a registry.
+
+### Events
+A Registry automation can watch for these events:
+- **Linking a new artifact to a collection**: Test and validate new models or datasets when they are registered.
```

Comment:
Generally a good idea to avoid using the product name as a verb.
```suggestion
- **Linking a new artifact to a collection**: Test and validate new models or datasets when they are added to a registry.
```